.github/workflows/build.yml

```
@@ -82,6 +82,18 @@ jobs:
           --health-interval 1s
           --health-timeout 5s
           --health-retries 60
+      consul:
+        image: docker.mirror.hashicorp.services/consul:latest
+        env:
+          CONSUL_LOCAL_CONFIG: "{\"acl\":{\"enabled\":true}}"
+        ports:
+        - 8500:8500
+        - 8600:8600
+        options: >-
+          --health-cmd "curl 'http://127.0.0.1:8500/v1/health/node/server-1'"
```

Think you want `curl -f` or `curl --fail` here since we want curl exit with a non-zero status whenever the response returns an error.

````suggestion
          --health-cmd "curl -f 'http://127.0.0.1:8500/v1/health/node/server-1'"
````